src/components/base/FormattedVal/index.js

```diff
@@ -143,4 +146,8 @@ function FormattedVal(props: Props) {
   )
 }
 
+FormattedVal.defaultProps = {
+  subMagnitude: 0,
```

Comment:
Is this useful?
`FormattedVal` already defaults `subMagnitude` to `0` IIRC

Reply:
If i don't set a default value at the component level, some use cases will forward `subMagnitude: undefined` to the helper func (formatCurrencyUnit) which will throw an error. The function's default values are set by an object merge which will not omit undefined values. This can be fixed by omitting the input's undefined values before merging with the defaults.

Reply:
(previously, `formatCurrencyUnit` wasn't using `subMagnitude` as a parameter in this context but i had to add it for this use case)